client/src/components/achievements/Achievement.js

```diff
@@ -123,4 +124,9 @@ const Achievement = ({ achievement }) => {
   );
 };
 
+// props validation
+Achievement.propTypes = {
+  achievement: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
Would it have saved you time by just supressing `react/forbid-prop-types` in .eslintrc.json since it's used quite a few times?

Reply:
Make sense... Will refactor them